project/AppDependencies.scala

```diff
@@ -1,31 +1,31 @@
-import sbt._
+import sbt.*
 
 object AppDependencies {
   import play.core.PlayVersion
 
-  val bootstrapVersion = "7.13.0"
+  val bootstrapVersion = "7.20.0"
 
-  val compile = Seq(
+  val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
     "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"             % "0.74.0",
-    "uk.gov.hmrc"       %% "play-frontend-hmrc"             % "6.4.0-play-28",
-    "uk.gov.hmrc"       %% "play-conditional-form-mapping"  % "1.12.0-play-28",
+    "uk.gov.hmrc"       %% "play-frontend-hmrc"             % "7.16.0-play-28",
+    "uk.gov.hmrc"       %% "play-conditional-form-mapping"  % "1.13.0-play-28",
     "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % bootstrapVersion,
-    "uk.gov.hmrc"       %% "domain"                         % "8.1.0-play-28"
+    "uk.gov.hmrc"       %% "domain"                         % "8.3.0-play-28"
   )
 
   val test: Seq[ModuleID] = Seq(
-    "org.scalatest"               %% "scalatest"                % "3.2.15",
+    "org.scalatest"               %% "scalatest"                % "3.2.16",
     "org.scalatestplus.play"      %% "scalatestplus-play"       % "5.1.0",
-    "org.jsoup"                   %  "jsoup"                    % "1.15.3",
+    "org.jsoup"                   %  "jsoup"                    % "1.16.1",
     "com.typesafe.play"           %% "play-test"                % PlayVersion.current,
     "org.scalatestplus"           %% "mockito-4-6"              % "3.2.15.0",
```

Comment:
both mockito deps can be replaced with this
`"org.scalatestplus"      %% "mockito-4-11"                   % "3.2.16.0",`